src/playground/auth/api-calls/api-calls.component.ts

```diff
@@ -0,0 +1,148 @@
+/**
+ * @license
+ * Copyright Akveo. All Rights Reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+import {Component, Inject} from '@angular/core';
+import { NbAuthResult, NbAuthService, NbAuthToken } from '@nebular/auth';
+import {HttpClient, HttpErrorResponse} from '@angular/common/http';
+import {catchError, map} from 'rxjs/operators';
+import { Wine } from './wine';
+import { Router } from '@angular/router';
+import { getDeepFromObject } from '@nebular/auth/helpers';
+import { NB_AUTH_OPTIONS } from '@nebular/auth';
+
+@Component({
+  selector: 'nb-playground-api-calls',
+  styles: [`
+    .rTable {
+      display: table;
+      width: 100%;
+    }
+    .rTableRow {
+      display: table-row;
+    }
+    .rTableHeading {
+      display: table-header-group;
+      background-color: #ddd;
+    }
+    .rTableCell, .rTableHead {
+      display: table-cell;
+      padding: 3px 10px;
+      border: 1px solid #999999;
+    }
+    .rTableHeading {
+      display: table-header-group;
+      background-color: #ddd;
+      font-weight: bold;
+    }
+    .rTableFoot {
+      display: table-footer-group;
+      font-weight: bold;
+      background-color: #ddd;
+    }
+    .rTableBody {
+      display: table-row-group;
+    }
+  `],
+  template: `
+    <router-outlet></router-outlet>
+    <nb-layout>
+      <nb-layout-column>
+        <nb-card>
+          <nb-card-body class="col-xl-4 col-lg-6 col-md-8 col-sm-12" style="margin:auto;">
```

Comment:
also would be really great if we can get rid of bootstrap as we are keen to remove it at some step